<?xml version="1.0"?>
<launch>

    <!-- <include file="$(find img_input)/launch/img_input.launch" /> -->
    <include file="$(find v9_ball_detector)/launch/v9_ball_detector.launch" />

    <node pkg="hline_square_detector" type="hline_square_detector_node" name="hline_square_detector_node" output="screen" >
<!--             <param name="variance" type="double" value="2.0" />     -->
        <remap from="/hline_square_detector_node/image_in" to="/img_input_node/image_out" />
        <!-- <remap from="/hline_square_detector_node/camera_info_in" to="/usb_cam/camera_info" /> -->
        <remap from="/hline_square_detector_node/frame_mode" to="/monitor_hline_square_node/frame_mode" />
        <remap from="/hline_square_detector_node/LUT_data" to="/monitor_hline_square_node/LUT_data" />
        <remap from="/hline_square_detector_node/save_param" to="/monitor_hline_square_node/save_param" />
        <remap from="/hline_square_detector_node/square_ref" to="/monitor_hline_square_node/square_ref" />
        <remap from="/hline_square_detector_node/line_tip" to="/v9_localization_node/line_tip" />
        <param name="ball_config_path" type="string" value="$(find hline_square_detector)/config/saved_config.yaml" />
    </node>
</launch>
